lunatrace/bsl/hasura/migrations/lunatrace/1659652941954_drop-old-vulnerability-table/down.sql

```diff
@@ -0,0 +1,2 @@
+SELECT * FROM public.identities;
+-- This down migration would be really difficult to create so it is omitted and this is treated as a one-way change.  We can discuss if this is acceptable
```

Comment:
I'm fine with this, IMO we should only migrate up anyway because most down migrations cause data loss.